__tests__/integration.test.js

```diff
@@ -61,6 +61,7 @@ describe("GET /api/articles/:article_id", () => {
         expect(article.article_img_url).toBe(
           "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
         );
+        expect(article.comment_count).toBe(11);
```

Comment:
we really want this in a new test as we dont want to change previous tests :)

Reply:
additional test created